source/plugins/projects/projects/index.js

```diff
@@ -1202,11 +1202,15 @@ let projects = {
 		let project = studio.settings.loadValue('projects', 'currentProject', null);
 		let file = studio.settings.loadValue('projects', 'currentFile', null);
 
-		if (project !== {} && project !== null) { 
-			if(await this.selectCurrentProject(project, false)) {
-				if (file !== {} && file !== null) {
-					await this.changeFile(project,file);
-				}
+		if (project !== {} && project !== null) {
```

Comment:
```suggestion
		if (project !== null) {
```
is project ever {} ?

Reply:
> Please take a look at the requested changes

Neither project nor file is ever {}, but while testing to see if they can be I've found out that closing and opening Wyliodrin Studio for the first time without creating a project, opens the program on a blank page.